extensions/npm/main.ts

```diff
@@ -1,17 +1,18 @@
 import { PhylumApi } from "phylum";
 
-// Parse CLI args.
-const args = Deno.args;
-if (args.length != 2 || args[0] !== "install") {
-    console.error("Usage: phylum npm install <PKG>");
+if (Deno.args.length >= 1 && 'install'.startsWith(Deno.args[0])) {
```

Comment:
Is the usage of `startsWith` here used to only cover `install` and the `i` shorthand, or are there other possible cases? Or does `npm` itself use this strategy?

Reply:
Npm uses the strategy. i, in, ins, inst, insta, instal, install are all valid. It's absolutely terrible, but someone must have spent too much time configuring cisco routers I guess.

Reply:
This doesn't cover all the other aliases: `add`, `isnt`, `isnta`, `isntal`, and `isntall`

Reply:
What the... why do those even exist?! I mean `add` sure, but `isntall`? Really?